configure.ac

```diff
@@ -114,6 +114,13 @@ AC_ARG_WITH(sim,
 	[with_sim=qemu]
 	)
 
+AC_ARG_WITH(languages,
+        [AS_HELP_STRING([--with-languages=c,c++,fortran],
+                [Sets the target languages, defaults to c,c++,fortran])],
+        [],
+        [with_languages=c,c++,fortran]
```

Comment:
As mentioned above, this seems to mean that `c,c++,fortran` will be enabled for **all** toolchain variants including bare-metal?

Reply:
> As mentioned above, this seems to mean that `c,c++,fortran` will be enabled for **all** toolchain variants including bare-metal?

oh yes because https://github.com/riscv-collab/riscv-gnu-toolchain/pull/1554#issuecomment-2366728731 says he needs fortran in default. I can remove fortran if you'd love to.

Reply:
Ideally this PR should allow the default language selection to be overridden (e.g. `--with-languages=c,c++,fortran,go`) but the existing defaults should remain as they are - i.e. `c,c++` for a bare-metal toolchain and `c,c++,fortran` for a Linux toolchain.

Reply:
> Ideally this PR should allow the default language selection to be overridden (e.g. `--with-languages=c,c++,fortran,go`) but the existing defaults should remain as they are - i.e. `c,c++` for a bare-metal toolchain and `c,c++,fortran` for a Linux toolchain. The comment that you linked to relates to the Linux toolchain(s).

yes now it does as you wish